next.config.js

```diff
@@ -2,8 +2,9 @@
 const withBundleAnalyzer = require('@next/bundle-analyzer')({
   enabled: process.env.ANALYZE === 'true',
 })
+const { withSentryConfig } = require('@sentry/nextjs')
```

Comment:
```suggestion

const { withSentryConfig } = require('@sentry/nextjs')
```

I'd like a newline here for readability. 😉

Reply:
I'm sorry but the spacing and sorting for the imports are set up with `eslint-plugin-import`. With our current config, it will group together (i.e. without newlines) all imports belonging to the same category. These two are external imports so they are group together:

<img width="708" alt="CleanShot 2021-07-12 at 20 30 35@2x" src="https://user-images.githubusercontent.com/2677072/125338066-0a9ecc00-e350-11eb-88ce-ad56d9e9fc71.png">

Reply:
Oh well. 😄 

Let's keep things as they are then. 😄